public/util/rendering/BatchQuadOverlayRenderer.js

```diff
@@ -214,7 +212,6 @@ export class BatchQuadOverlayRenderer extends WebGLOverlayRenderer {
     const cell = plot.cell;
     const proj = this.getOrthoMatrix();
     const scale = Math.pow(2, plot.zoom - cell.zoom);
-    const opacity = this.overlay.opacity;
```

Comment:
Why removing the opacity? I think it makes it harder to know where the buckets are, specially in a use case where a whole region could be covered.

Reply:
This is a const opacity that was being applied to every quad. The quads still support opacity in their VBO. Essentially what was happening in the shader was vbo.alpha * overlay.opacity. <- so the overlay.opacity is kind of redundant since we can control the opacity through the vertex data.